src/Microsoft.AspNet.Mvc.Core/Controller.cs

```diff
@@ -618,6 +618,119 @@ public virtual HttpNotFoundResult HttpNotFound()
         }
 
         /// <summary>
+        /// Creates an <see cref="BadRequestResult"/> that produces a Bad Request (400) response.
+        /// </summary>
+        /// <returns>The created <see cref="BadRequestResult"/> for the response.</returns>
+        [NonAction]
+        public virtual BadRequestResult HttpBadRequest()
+        {
+            return new BadRequestResult();
+        }
+
+        /// <summary>
+        /// Creates a <see cref="CreatedResult"/> object that produces a Created (201) response.
+        /// </summary>
+        /// <param name="uri">The URI at which the content has been created.</param>
+        /// <param name="value">The content value to negotiate and format in the entity body.</param>
+        /// <returns>The created <see cref="CreatedResult"/> for the response.</returns>
+        [NonAction]
+        public virtual CreatedResult Created([NotNull] string uri, object value)
+        {
+            return new CreatedResult(uri, value);
+        }
+
+        /// <summary>
+        /// Creates a <see cref="CreatedResult"/> object that produces a Created (201) response.
+        /// </summary>
+        /// <param name="uri">The URI at which the content has been created.</param>
+        /// <param name="value">The content value to negotiate and format in the entity body.</param>
```

Comment:
For all of these just say "The content value to format in the entity body." (drop the 'to negotiate').

The reason is that saying it doesn't add much clarity, and it would take several sentences to really explain. We don't explain elsewhere that conneg is part of the system, it just IS.